src/Api/Cloud.php

```diff
@@ -196,7 +196,7 @@ public function getEncodings(array $filter = array())
     /**
      * {@inheritDoc}
      */
-    public function getEncodingsWithStatus($status, array $filter = array())
+    public function getEncodingsWithStatus($status, array $filter = null)
     {
         $filter['status'] = $status;
```

Comment:
Well, it works (see https://3v4l.org/p1lU1) at least for PHP versions <= 7.1. But I agree that this is not a very clean solution. So I pushed some more code making sure that `$filter` is an array before modifying it.